docs/source/config_files.rst

```diff
@@ -95,6 +95,27 @@ version of Ubuntu.
 We use ``apt.txt``, for example, to install LaTeX in our
 `example apt.txt for LaTeX <https://github.com/binder-examples/latex/blob/master/apt.txt>`_.
 
+``setup.py``
```

Comment:
How did this get here? Just to make sure this isn't going to cause a merge conflict or overwrite edits  recently made in master

Reply:
whops, my bad - i screwed up the merge conflict and thought this needs to stay.